apps/snitch_core/priv/repo/seed/users.ex

```diff
@@ -19,7 +19,7 @@ defmodule Snitch.Seed.Users do
       user("Harry", "Potter", "admin@aviacommerce.com", @admin_passwd, true, admin_role.id)
     ]
 
-    Repo.insert_all(User, users, on_conflict: :nothing, conflict_target: [:email])
+    Repo.insert_all(User, users, on_conflict: :nothing, conflict_target: [:email, :deleted_at])
```

Comment:
@jyotigautam need a good explanation for this.

Reply:
conflict target looks up for the columns where conflicts can occur, if not given explicitly, it is by default made of primary keys and/or unique constraints. Here, if I had not mentioned the conflict_target fields, then also it would have picked [:email, :deleted_at] fields according to the database constraint on user table.